packages/in_app_purchase/in_app_purchase_platform_interface/lib/src/types/query_purchase_details_response.dart

```diff
@@ -0,0 +1,26 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'in_app_purchase_error.dart';
+import 'purchase_details.dart';
+
+/// The response object for fetching the past purchases.
+///
+/// An instance of this class is returned in [InAppPurchaseConnection.queryPastPurchases].
+class QueryPurchaseDetailsResponse {
```

Comment:
Is this class still needed?

Reply:
No it is not needed anymore. I removed it.